docs/workflows/sharing.md

```diff
@@ -22,13 +22,13 @@ Users can share workflows they created. Instance owners, and users with the admi
 
 ## View shared workflows
 
-You can browse and search workflows on the **Workflows** list.
+You can browse and search workflows on the **Workflows** list. The workflows in the list depend on the project:
 
-* **All workflows** lists all workflows you can access. This includes:
+* **Home** lists all workflows you can access. This includes:
 	* Your own workflows.
 	* Workflows shared with you.
-	* If you are logged in as instance owner: all workflows in the instance.
-* **My workflows**: workflows you created.
+	* If you log in as the instance owner or admin: all workflows in the instance.
```

Comment:
You would also see workflows and credentials that belong to projects you are part of in your home screen